content/kb/space-consumption-guideline.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Space consumption guideline"
+author: Shuo Wu
+draft: false
+date: 2023-08-25
+categories:
+- "instruction"
+---
+
+## Applicable versions
+
+All Longhorn versions, but some features are introduced in v1.4.0 or v1.5.0
+
+## Volumes consume much more space than expected
+
+Due to the fact that Longhorn volumes can hold historic data as snapshots, the volume actual size can be much greater than the spec size. For more details, you can check [this session](../../docs/1.5.1/volumes-and-nodes/volume-size/#volume-actual-size) for a better understanding over the concept of volume size.
+
+Besides, some operations like backup, rebuilding, or expansion, will lead to a hidden system snapshot creation. Hence there may be some snapshots even if users never create a snapshot for a volume manually.
+
+To eliminate space being wasted the historic data/snapshots, we would recommend applying a recurring job like `snapshot-delete` that limits the snapshot counts of volumes. You can check [the recurring job session](../../docs/1.5.1/snapshots-and-backups/scheduling-backups-and-snapshots) and see how to work.
+
+## Filesystem used size is much smaller than volume actual size
+
+The reason for this symptom is explained in [the volume size session](../../docs/1.5.1/volumes-and-nodes/volume-size/#volume-actual-size) as well. Briefly say, a volume is a block device, which does not recognize the filesystem. In other words, deleting a file from the filesystem does not mean that the related blocks can be released by the block device.
```

Comment:
NIT: I had to read these sentences a couple of times to get the meaning.

```suggestion
The reason for this symptom is explained in [the volume size session](../../docs/1.5.1/volumes-and-nodes/volume-size/#volume-actual-size) as well. Briefly, a Longhorn volume is a block device which does not recognize the filesystem used on top of it. Deleting a file is a filesystem layer operation that does not actually free up blocks from the underlying volume.
```

Reply:
Oh sorry for the confusion. Your statement is definitely easy to understand.